default/python/common/configure_logging.py

```diff
@@ -158,7 +158,9 @@ def __init__(self, level):
 
     def emit(self, record):
         msg = self.format(record)
-        self.logger(msg, str(record.filename), record.lineno)
+        if not msg.endswith("\n"):
```

Comment:
IMHO it will be nice to leave a comment in the code about the reason.

Reply:
https://github.com/freeorion/freeorion/commit/e405949c3a44d0b06d5f93e521dead5e361537c8